pkg/dependenciesdistributor/dependencies_distributor.go

```diff
@@ -336,9 +338,11 @@ func (d *DependenciesDistributor) ReconcileResourceBinding(key util.QueueKey) er
 
 	dependencies, err := d.ResourceInterpreter.GetDependencies(workload)
 	if err != nil {
+		d.EventRecorder.Eventf(workload, corev1.EventTypeWarning, workv1alpha2.EventReasonGetDependenciesFailed, "Get dependencies failed, err: %s.")
 		klog.Errorf("Failed to customize dependencies for %s(%s), %v", workload.GroupVersionKind(), workload.GetName(), err)
 		return err
 	}
+	d.EventRecorder.Eventf(workload, corev1.EventTypeNormal, workv1alpha2.EventReasonGetDependenciesSucceed, "Get dependencies(%v) succeed.", dependencies)
```

Comment:
Please don't just print with the default format,

Reply:
%#:
Get dependencies([{%!)(string=v1) %!)(string=ConfigMap) %!)(string=default) %!)(string=my-nginx-config)}])
%+v:
Get dependencies([{APIVersion:v1 Kind:Config Namespace:default Name:test}])

Reply:
/cc @RainbowMango. Do you think the latter format is ok?

Reply:
sounds good to me.